commands.py

```diff
@@ -0,0 +1,102 @@
+from SchemaKeys import SchemaKeys
+import json, ast
+import os
+
+parent_dir = os. getcwd() 
+
+class ICommand():
+    def execute(self):
+        pass
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):
+        self._schema = schema
+
+    def execute(self):
+        f = open(self._schema, "r")
+        data = json.load(f)
+        dir = data[SchemaKeys().DATABASE]
+        path = os.path.join(parent_dir, dir)
+
+        if not (os.path.exists(path)):
+            os.mkdir(path)
+
+        for table in data[SchemaKeys().TABLES]:
+            t_path = os.path.join(path, table[SchemaKeys().NAME])
+            if not (os.path.exists(t_path)):
+                os.mkdir(t_path)
+
+class CommandFactory:
+    def create(self, args):
+        cmd = args.command.lower()
```

Comment:
What's the reason for lower function, Not a user input

Reply:
No, it is a user input.

Reply:
Besides, we have to handle any logical error that may occur